app/views/catalog/_metadata_block.html.erb

```diff
@@ -4,6 +4,15 @@
   <div class="<%= presenter_class %> card mb-2 rounded-0">
     <div class="card-header rounded-0"><%= title %></div>
     <div class="card-body">
+      <% if presenter == AboutThisItemPresenter && document['visibility_ssi'] == 'low_res' %>
```

Comment:
For our metadata blocks, we currently have a pattern where if a given block needs extra configuration (e.g., the access and copyright block, the this item contains block), we make a new file for that block and customize the behavior there. Instead of checking for the presenter every time, it possible to reroute this functionality into an `about_this_item_block.html.erb` file?

Reply:
Sure. I'll jump on that.